signature-jwt.go

```diff
@@ -58,12 +69,24 @@ func (jwt *JWT) GenerateToken(userName string) (string, error) {
 }
 
 // Authenticate - authenticates incoming username and password.
-func (jwt *JWT) Authenticate(userName, password string) bool {
-	userName = strings.TrimSpace(userName)
-	password = strings.TrimSpace(password)
+func (jwt *JWT) Authenticate(userName, password string) error {
+	if !isValidAccessKey.MatchString(userName) {
+		return fmt.Errorf("invalid user name")
```

Comment:
> you need to keep this inside.
> 
>    if !isValidAccessKey.MatchString(args.AccessKey) {
>                 return &json2.Error{Message: "Invalid Access Key"}
>         }
>         if !isValidSecretKey.MatchString(args.SecretKey) {
>                 return &json2.Error{Message: "Invalid Secret Key"}
>         }

Its not needed since this a method of `*JWT`